### PR TITLE
SW-1136-cancelling-a-laser-job-message-not-clear-enough

### PR DESCRIPTION
**Pull request by JFSolution**
- changed cancel modal to talk about cancel laser job instead of print
- added translation

### CODE REVIEW

File: octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1324,39 +1347,65 @@ msgstr ""
 "sie wieder startet."
 
 #: octoprint_mrbeam/static/js/user_notification_viewmodel.js:55
+msgid "No update information"
+msgstr ""
```
Comment (khaledsherkawi) [inline octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po]: Why are these added here in this ticket?

File: octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -5720,6 +5724,26 @@ msgstr "Daemon offline"
 msgid "Reset daemon"
 msgstr "Dämon zurücksetzen"
 
+#~ msgid "QuickstartGuide_en.pdf"
```
Comment (khaledsherkawi) [inline octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po]: Is there a reason there are removed here? Are we doing the translation for multiple tickets?

File: translations/messages.pot [hunk 3]
```diff
@@ -1138,46 +1161,71 @@ msgid "The camera was stopped recently, it will take a few seconds to restart."
 msgstr ""
 
 #: octoprint_mrbeam/static/js/user_notification_viewmodel.js:55
-msgid "Camera Error"
+msgid "No update information"
 msgstr ""
 
 #: octoprint_mrbeam/static/js/user_notification_viewmodel.js:56
 msgid ""
+"No information about available updates could be retrieved, please try "
```
Comment (khaledsherkawi) [inline translations/messages.pot]: There as well

File: octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po (hunk 1 above)
Comment (JFSolution) [reply octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po]: Good catch, I will revert the translation and add it to a new ticket

**Review by khaledsherkawi — APPROVED** [review]
